pkg/report/output/dataflow/dataflow.go

```diff
@@ -0,0 +1,83 @@
+package dataflow
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/bearer/curio/pkg/report/detections"
+	"github.com/bearer/curio/pkg/report/output/dataflow/datatypes"
+	"github.com/bearer/curio/pkg/report/output/dataflow/risks"
+
+	"github.com/bearer/curio/pkg/report/output/dataflow/types"
+)
+
+type DataFlow struct {
+	Datatypes []types.Datatype     `json:"data_types,omitempty"`
+	Risks     []types.RiskDetector `json:"risks,omitempty"`
+}
+
+var allowedDetections []detections.DetectionType = []detections.DetectionType{detections.TypeSchema, detections.TypeSchemaClassified, detections.TypeCustom, detections.TypeCustomClassified}
+
+func GetOuput(input []interface{}) (*DataFlow, error) {
+	dataTypesHolder := datatypes.New()
+	risksHolder := risks.New()
+
+	for _, detection := range input {
+		detection, ok := detection.(map[string]interface{})
+		if !ok {
+			return nil, fmt.Errorf("found detection in report which is not object")
+		}
+
+		detectionTypeS, ok := detection["type"].(string)
+		if !ok {
+			continue
+		}
+
+		detectionType := detections.DetectionType(detectionTypeS)
+
+		isDataflow := false
+		for _, allowedDetection := range allowedDetections {
+			if (detectionType) == allowedDetection {
+				isDataflow = true
+			}
+		}
+
+		if !isDataflow {
+			continue
+		}
+
+		var castedDetection detections.Detection
```

Comment:
Suggestion to change the variable name: `cast` reads more naturally to me (unless `casted` is a Go convention, in which case ignore me :) )

```suggestion
		var castDetection detections.Detection
```